libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1497,6 +1497,11 @@ bool AP_GPS::calc_blend_weights(void)
         if (get_rate_ms(i) > max_rate_ms) {
             max_rate_ms = get_rate_ms(i);
         }
+        if (isinf(state[i].speed_accuracy) ||
```

Comment:
We'll fail even if the blend mask specifies not to use the speed accuracy?

Reply:
we could separate it out with more complex logic, but if a GPS is giving infinity for any accuracy I don't want to use it in blending

Reply:
Also, if we can stomach working with inf's, then we should probably be checking `speed_accuracy_sum_sq` rather than the input - it might go inf even if this input isn't.

Reply:
yes, I don't think we want to do blending if we have infinite values for any accuracy